mono/mini/mini.c

```diff
@@ -3499,6 +3510,11 @@ mini_method_compile (MonoMethod *method, guint32 opts, MonoDomain *domain, JitFl
 	MONO_TIME_TRACK (mono_jit_stats.jit_method_to_ir, i = mono_method_to_ir (cfg, method_to_compile, NULL, NULL, NULL, NULL, 0, FALSE));
 	mono_cfg_dump_ir (cfg, "method-to-ir");
 
+	if (cfg->gdump_ctx != NULL) {
+		mono_insert_nop_in_empty_bb (cfg);
```

Comment:
it's a hack.  IGV (the graph visualization tool) doesn't like empty basic blocks.